### ISSUE TITLE
qlstephen on clean Catalina install

### ISSUE THREAD

**Issue by ptoche** [whomwah/qlstephen#98]
I used to have qlstephen on my previous (unclean) install of Catalina. Now that I've clean installed it, I'm not sure if it's working at all. For instance, I can't preview text files with extension `.in`, `.ncx`, `.opf`, to name a few. I can preview many other types though. 

Here's what I did. Installed the latest `QLStephen.qlgenerator` in `~/Library/QuickLook` and tried also in `/Library/QuickLook`. I ran `qlmanage -r` and `qlmanage - m`. I haven't edited any files inside `Contents` (`InfoPlist.strings` is empty, if that matters). 

The first few lines of `qlmanage -m` look like this:

     qlmanage -m
    server: living for 0s (0 requests handled) - instant off: yes - arch: X86_64 - user id: 501
    memory used: 0 MB (545296 bytes) - used descriptors: 19/256
    plugins:
      com.apple.addressbook.group -> /System/Library/QuickLook/Contact.qlgenerator (906.2)
      org.tukaani.xz-archive -> /Applications/BetterZip.app/Contents/Library/QuickLook/BetterZipQL.qlgenerator (1466)
      com.apple.iconset -> /System/Library/QuickLook/Icon.qlgenerator (906.2)
    ... and many more lines

**Comment by chrisjsewell** [comment]
Same here. I installed on a new macbook with 10.15.5, *via* homebrew, and I don't see any previews available e.g for `.yml`

**Comment by flemingm** [comment]
check if it code signed, if it not it will not load on 10.15.5.

qlmanage -m | grep QLS
codesign -dvvvv  /Library/QuickLook/QLStephen.qlgenerator/

**Comment by ptoche** [comment]
Thanks for the feedback. Here's what I get:

    ~ qlmanage -m|grep QLS
      public.data -> /Library/QuickLook/QLStephen.qlgenerator (1.5.1)
      public.content -> /Library/QuickLook/QLStephen.qlgenerator (1.5.1)

    ~ codesign -dvvvv /Library/QuickLook/QLStephen.qlgenerator/
    Executable=/Library/QuickLook/QLStephen.qlgenerator/Contents/MacOS/QLStephen
    Identifier=com.whomwah.quicklookstephen
    Format=bundle with Mach-O thin (x86_64)
    CodeDirectory v=20100 size=717 flags=0x2(adhoc) hashes=17+3 location=embedded
    VersionPlatform=1
    VersionMin=656896
    VersionSDK=659200
    Hash type=sha256 size=32
    CandidateCDHash sha1=c5e11076fc642cd56e71da96fc543afa5c958c61
    CandidateCDHashFull sha1=c5e11076fc642cd56e71da96fc543afa5c958c61
    CandidateCDHash sha256=1e0e19014012aa2bf29d42ea811e2379e431bcaf
    CandidateCDHashFull sha256=1e0e19014012aa2bf29d42ea811e2379e431bcaf8f922c10c1f97d6fc004fb49
    Hash choices=sha1,sha256
    CMSDigest=317f3713bf1125d9667d7563ab84f3a58e019c06e808317ddfbb89b477116080
    CMSDigestType=2
    Page size=4096
    CDHash=1e0e19014012aa2bf29d42ea811e2379e431bcaf
    Signature=adhoc
    Info.plist entries=29
    TeamIdentifier=not set
    Sealed Resources version=2 rules=13 files=1
    Internal requirements count=0 size=12

**Comment by welby12** [comment]
Having the same issue on macOS 10.15.6 via Homebrew. This is the message I was getting on boot. I was able to fix by running this command, which I have used for other improperly signed stuff

`xattr -d com.apple.quarantine ~/Library/QuickLook/QLStephen.qlgenerator`
<img width="532" alt="Screen Shot 2020-07-28 at 6 12 38 PM" src="https://user-images.githubusercontent.com/13987200/88745460-fc37b000-d0fe-11ea-87db-36d52740c538.png">

**Comment by fuzunspm** [comment]
> `xattr -d com.apple.quarantine ~/Library/QuickLook/QLStephen.qlgenerator`

this won't work on my side:

```
xattr: /Users/username/Library/QuickLook/QLStephen.qlgenerator: No such xattr: com.apple.quarantine
```

**Comment by theRealPadster** [comment]
@fuzunspm I get the same "no such xattr" when I run that, but if I go inside the folder and run it on the executable, it doesn't give me an error. I haven't given it much time to see if it fixes the popup, but maybe give this a shot?

**Comment by DKroot** [comment]
`xattr -d com.apple.quarantine ~/Library/QuickLook/QLStephen.qlgenerator` worked like a charm on my Catalina 10.5.7.

**Comment by theRealPadster** [comment]
And just an update on my attempt with `xattr -d com.apple.quarantine ~/Library/QuickLook/QLStephen.qlgenerator/Contents/MacOS/QLStephen`
It did not work.

**Comment by DKroot** [comment]
Since `xattr -d` *removes* the attribute, it could be done only once. If you get `No such xattr` message, it means that it has been already done and if the issue still occurs, the problem has some other cause.

**Comment by theRealPadster** [comment]
Fair point. I just thought it might have been re-added with some update or something.
